static/app/views/alerts/incidentRules/incidentRulePresets.tsx

```diff
@@ -49,6 +49,8 @@ export function makeDefaultCta({
     };
   }
 
+  const transactionFields = ['title', 'count()', 'count_unique(user)'];
```

Comment:
we declare the other array here, can we move this logic to the same place https://github.com/getsentry/sentry/blob/4d8d544c8b195103a9f0474786f27dba3fef8ae4/static/app/views/alerts/rules/details/metricChart.tsx#L309